code/api/src/modules/user_product/model.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
```

Comment:
Joins table migration. The kept quantity will be used to determine if the user kept the product or not